tests/large/test_ide.py

```diff
@@ -80,6 +80,106 @@ def test_default_eclipse_ide_install(self):
         self.wait_and_close()
 
 
+class EclipsePHPIDETests(LargeFrameworkTests):
```

Comment:
This class is exactly the same than EclipseJavaIDETests with few differences:
installed_path, desktop_filename, command and the "is already installed" (eclipse-php) in umake.

I think we can avoid this duplication by setting up all those parameters in setUp() (you already have two of them) of EclipseJavaIDETests, then inheriting from it and change changing setUp() to match the new php and cpp class, thoughts?